test/consistency.jl

```diff
@@ -0,0 +1,95 @@
+using Test
+using CondaPkg
+using PythonCall
+
+# Load the Python module/package
+numpy = pyimport("numpy")
+mrcfile = pyimport("mrcfile")
+
+@testset "Consistency Test" begin
+    function map_test(map_jl::Array{Float32,3}, map_py::Array{Float32,3})
```

Comment:
I prefer we define all of these functions outside of the `@testset` and give them more descriptive names.